libs/core/pyproject.toml

```diff
@@ -15,7 +15,7 @@ langsmith = "^0.1.0"
 tenacity = "^8.1.0"
 jsonpatch = "^1.33"
 PyYAML = ">=5.3"
-packaging = "^23.2"
+packaging = ">=23.2,^24.0"
```

Comment:
>=23.2 implies 24 and 25 if i'm not mistaken?

Reply:
`>=` equal or higher than.

AFAIK this implies higher than or equal to 23.2 and lower than 25. 

Ref: https://python-poetry.org/docs/dependency-specification/

Reply:
afaik the `,` indicates a logical AND not a logical OR, so `>=23.2,^24.0` ends up just being `^24.0`. should it isntead be `>=23.2,<25`?